app/src/main/java/com/douglasbruce/tasky/features/event/EventViewModel.kt

```diff
@@ -19,7 +19,7 @@ class EventViewModel @Inject constructor(
     private val eventArgs: EventArgs = EventArgs(savedStateHandle)
 
     var state by savedStateHandle.saveable {
-        mutableStateOf(EventState(id = eventArgs.eventId))
+        mutableStateOf(EventState(id = if(eventArgs.eventId == "-1") null else eventArgs.eventId))
```

Comment:
Does the Compose nav framework support nullable Strings? I always find it better to use nulls to indicate that a value doesn't exist (which is the exact purpose of them) rather than something like `"-1"` which you'd always need to look up in code if you forgot/are new in the team. With nullable strings, it could simply look like this: `id = eventArgs.eventId`